src/modules/lreproxy/lreproxy.h

```diff
@@ -0,0 +1,120 @@
+/*
+ * Copyright (C) 2019-2020 Mojtaba Esfandiari.S, Nasim-Telecom
```

Comment:
From brief look to it, this is pretty much carboon-copy of the rtpproxy module, as such original (c) line of the code should be reproduced. Thanks!